url.bs

```diff
@@ -2247,6 +2247,18 @@ and then runs these steps:
      <li><p>If <a>c</a> is an <a>ASCII alphanumeric</a>, U+002B (+), U+002D (-), or U+002E (.),
      append <a>c</a>, <a lt="ASCII lowercase">lowercased</a>, to <var>buffer</var>.
 
+     <li>
+      <p>Otherwise, if <a>c</a> is U+003A (:) and <var>buffer</var>'s <a for=string>length</a> is 1 and 
+      <var>buffer</var> contains only an <a>ASCII alpha</a> and <a>remaining</a> starts with U+005C (\), then:
```

Comment:
```suggestion
      <p>Otherwise, if <a>c</a> is U+003A (:), <var>buffer</var>'s <a for=string>length</a> is 1, 
      <var>buffer</var> contains only an <a>ASCII alpha</a>, and <a>remaining</a> starts with
      U+005C (\):
```